src/Geometry/axistensors.jl

```diff
@@ -269,10 +269,30 @@ Base.propertynames(x::AxisVector) = symbols(axes(x, 1))
     end
 end
 
+const AdjointAxisTensor{T, N, A, S} = Adjoint{T, AxisTensor{T, N, A, S}}
+
+Base.show(io::IO, a::AdjointAxisTensor{T, N, A, S}) where {T, N, A, S} =
+    print(io, "adjoint($(a'))")
+
+components(a::AdjointAxisTensor) = components(parent(a))'
+
+Base.zero(a::AdjointAxisTensor) = zero(a')'
+Base.zero(::Type{AdjointAxisTensor{T, N, A, S}}) where {T, N, A, S} =
+    zero(AxisTensor{T, N, A, S})'
+
+@inline +(a::AdjointAxisTensor) = (+a')'
+@inline -(a::AdjointAxisTensor) = (-a')'
+@inline +(a::AdjointAxisTensor, b::AdjointAxisTensor) = (a' + b')'
+@inline -(a::AdjointAxisTensor, b::AdjointAxisTensor) = (a' - b')'
+@inline *(a::Number, b::AdjointAxisTensor) = (a * b')'
+@inline *(a::AdjointAxisTensor, b::Number) = (a' * b)'
+@inline /(a::AdjointAxisTensor, b::Number) = (a' / b)'
+@inline \(a::Number, b::AdjointAxisTensor) = (a' \ b)'
+
+@inline (==)(a::AdjointAxisTensor, b::AdjointAxisTensor) = a' == b'
```

Comment:
I'd like for us to add tests in the test suite for confirming the correct return type.